src/mesh/mesh.cpp

```diff
@@ -2168,9 +2168,9 @@ void Mesh::ReserveMeshBlockPhysIDs() {
   }
   if (CR_ENABLED) {
     ReserveTagPhysIDs(CellCenteredBoundaryVariable::max_phys_id);
-  if (CRDIFFUSION_ENABLED)
-    ReserveTagPhysIDs(1);
-
+    if (CRDIFFUSION_ENABLED)
```

Comment:
@tomidakn after seeing `outputs.cpp`, I think I messed this up. Should this conditional be nested within `CR_ENABLED` or not?

Reply:
It should not be nested. CR and CRDIFFUSION do not share any component.